packages/rsync.rb

```diff
@@ -30,16 +30,7 @@ class Rsync < Package
   no_patchelf
   no_zstd
 
-  def self.build
-    system "./configure #{CREW_OPTIONS} --disable-maintainer-mode"
-    system 'make'
-  end
+  configure_options '--disable-maintainer-mode'
```

Comment:
It seems like maintainer mode is disabled already (lines 122-126 of the configure.ac file in the prvoided tarball):
```
AC_ARG_ENABLE(maintainer-mode,
    AS_HELP_STRING([--enable-maintainer-mode],[enable to turn on extra debug features]))
if test x"$enable_maintainer_mode" = x"yes"; then
    CFLAGS="$CFLAGS -DMAINTAINER_MODE"
fi
```